src/lib/Libpbs/Makefile.am

```diff
@@ -36,25 +36,13 @@
 # trademark licensing policies.
 #
 
-lib_LTLIBRARIES = libpbs.la
+lib_LIBRARIES = libpbs.a
```

Comment:
How will libtool behave with this change? We want to build both static and dynamic versions of the library. Do we still get both with this change? More info here... https://www.gnu.org/software/automake/manual/html_node/A-Shared-Library.html#A-Shared-Library